GoogleForCreators/web-stories-wp
# Add grouping to templates

## Context


Add grouping, group names, and layer locking to templates
## Summary


Adds grouping, group names, and layer locking to templates
## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Go to Explore Templates
2. Click though the templates
- Notice most templates (that seem to need it, based on amount of layers - about 10 layers or more) now utilize layer grouping and locking.


## Reviews

### Does this PR have a security-related impact?


No
### Does this PR change what data or activity we track or use?


No
### Does this PR have a legal-related impact?


No
## Checklist




---



Fixes #11697
